src/main/drivers/vtx_soft_spi_rtc6705.c

```diff
@@ -123,7 +123,21 @@ void rtc6705_soft_spi_set_channel(uint16_t channel_freq)
 
 void rtc6705_soft_spi_set_rf_power(uint8_t reduce_power)
 {
-    rtc6705_write_register(7, (reduce_power ? (PA_CONTROL_DEFAULT | PD_Q5G_MASK) & (~(PA5G_PW_MASK | PA5G_BS_MASK)) : PA_CONTROL_DEFAULT));
+    switch(reduce_power)
+    {
+        case 0:// 1dBm
+            rtc6705_write_register(7, 0x006A);
```

Comment:
I also have other rct6705 based modules I can test with this weekend. tx5813 (25mW), tx5823 (200mW), fx758 (200mW), and LaForge VTX (25mW, 200mW, 400mW). 
Unfortunately I don't have an FC with integrated VTX to test full flow at the moment but I can get one on order if needed.